packages/core/lib/Mediator.ts

```diff
@@ -112,3 +112,6 @@ export interface IMediatorArgs<A extends Actor<I, T, O>,
    */
   bus: Bus<A, I, T, O>;
 }
+
+export type Mediate<I extends IAction, O extends IActorOutput, T extends IActorTest = IActorTest> =
```

Comment:
Any particular reason why the order is I-O-T instead of the usual I-T-O?
If not, I would favor using the latter for consistency.

Reply:
`T` can often be defaulted to `IActorTest` e.g. `export type MediatorInit = Mediate<IActionInit, IActorOutputInit>;` - and default parameters must come after required parameters.

Reply:
Ah, I missed the default param definition.
Ok, good enough a reason for me :-)